createrelease.rb

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env ruby
```

Comment:
Ox flirting with Clojure and you with Ruby?  You guys on a bet, or something? :P